### DISCUSSION TITLE
How to replace Colon with Space or dash

### DISCUSSION THREAD

**Discussion by qqzzxx** [Flexget/Flexget#3871] [Q&A]
like imdb_name, for some movies there is a : Colon in title, which : is not allowed in Windows filename,

how can i replace the colon with just Space or dash.

**Comment by BrutuZ** [comment]
https://flexget.com/Plugins/manipulate

**Comment by gazpachoking** [comment]
There is also the pathscrub plugin specifically for this. If you put it in to windows mode, it will make sure created filenames are windows compatible. https://flexget.com/Plugins/pathscrub
